client/src/services/Fetch.ts

```diff
@@ -38,7 +38,7 @@ class Fetch {
   private fetch<T>(options: AxiosRequestConfig): Promise<T> {
     return axios(options)
       .then((res) => {
-        return res.data;
+        return res.data.data;
```

Comment:
not so good, maybe use `.then(({ data: resData }) => {` to avoid confusion in namings `data.data`